Gemfile

```
@@ -2,9 +2,10 @@ source "https://rubygems.org"
 gemspec
 
 gem 'rails', '~> 5.0.0'
+gem 'minitest-rails', '~> 5.0.0'
 
 group :active_record do
-  gem 'sqlite3', '~> 1.3.0'
+  gem 'sqlite3', "~> 1.3.0"
```

This seems more consistent with the rest of the file:

```suggestion
  gem 'sqlite3', '~> 1.3.0'
```